composer.json

```diff
@@ -16,9 +16,7 @@
     ],
     "require": {
         "php": ">=7.2",
-        "illuminate/database": "~5.5|~6|~7|~8",
-        "illuminate/http": "~5.5|~6|~7|~8",
```

Comment:
Why require the entire framework?

Reply:
#44 requested it, that's all. I don't see any reason not to do that on all our packages. They are being used by Laravel projects, so there's no advantage to requiring only the individual parts.

Reply:
I don't think that is the conflict.  I think there is a different conflict in that person's project.  It just means that we are downloading & pulling in way more packages into the project for when we test & such, so I would prefer to be explicit about the packages being used instead of relying on the entire kitchen sink.